src/Core/Managed/Net40/Extensibility/Implementation/Platform/PlatformImplementation.cs

```diff
@@ -29,32 +22,47 @@ public string ReadConfigurationXml()
             // Config file should be in the base directory of the app domain
             string configFilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "ApplicationInsights.config");
 
-            // Ensure config file actually exists
-            if (File.Exists(configFilePath))
+            try
             {
-                return File.ReadAllText(configFilePath);
+                // Ensure config file actually exists
+                if (File.Exists(configFilePath))
+                {
+                    return File.ReadAllText(configFilePath);
+                }
+            }
+            catch (FileNotFoundException)
```

Comment:
Yes, you got it right

---
In reply to: [77934014](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/304#discussion_r77934014) [](ancestors = 77934014)